assets/js/details.js

```diff
@@ -0,0 +1,34 @@
+  const params = new URLSearchParams(window.location.search);
+  const movieId = params.get("id");
+  const apiKey = "9f120f0e95e5cedf05a78abda9ef8477";
+  
+  async function getMovieDetails () {
+   const result = await fetch(`https://api.themoviedb.org/3/movie/${movieId}?api_key=${apiKey}`);
+   const resultJson = await result.json();
+   const {results: movie} = resultJson;
```

Comment:
you can skip defining `resultJson`.

Reply:
Also destructure whatever is necessary here annd use it there (`listHtml`).

Reply:
Done.